src/analysis/normed/group/pointwise.lean

```diff
@@ -80,4 +80,25 @@ lemma closed_ball_zero_add_singleton (x : E) (r : ℝ) :
   closed_ball 0 r + {x} = closed_ball x r :=
 by simp
 
+lemma is_compact.cthickening_eq_add_closed_ball
+  {s : set E} (hs : is_compact s) {r : ℝ} (hr : 0 ≤ r) :
+  cthickening r s = s + closed_ball 0 r :=
```

Comment:
Could you please extract a lemma for metric spaces,
```lean
lemma is_compact.cthickening_eq_add_closed_ball' {E : Type*} [pseudo_metric_space E]
  {s : set E} (hs : is_compact s) {r : ℝ} (hr : 0 ≤ r) :
  cthickening r s = ⋃ x ∈ s, closed_ball x r :=
```

Reply:
Done (together with a few other lemmas to flesh out the API)